app/javascript/react/containers/ParkShowContainer.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react'
+
+class ParkShowContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      parkName: '',
+      parkAddress: '',
+      parkCity: '',
+      parkState: '',
+      parkZip: '',
+      parkDescription: ''
+    }
+  }
+
+  componentDidMount() {
+    fetch(`/api/v1/parks/${this.props.params.id}`)
+    .then(response => {
+      if (response.ok) {
+        return response;
+      } else {
+        let errorMessage = `${response.status} (${response.statusText})`,
+            error = new Error(errorMessage);
+        throw(error);
+      }
+    })
+    .then(response => response.json())
+    .then(response => {
```

Comment:
if we restructure the state management in this component, we can instead do:

`this.setState({park: response.park})`